modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/model/AssetVocabularyMetric.java

```diff
@@ -82,6 +95,10 @@ public int hashCode() {
 		return Objects.hash(_assetCategoryMetrics);
 	}
 
+	public boolean isEmpty() {
```

Comment:
Very confusing that the method isEmpty return another thing other than this instance != _EMPTY.

Reply:
true! changing it ;)

Reply:
I just got rid of it ;)